ocs_ci/ocs/ocs_upgrade.py

```diff
@@ -123,28 +124,15 @@ def verify_image_versions(old_images, upgrade_version, version_before_upgrade):
     )
     verify_pods_upgraded(old_images, selector=constants.MDS_APP_LABEL, count=2)
     if config.ENV_DATA.get("platform") in constants.ON_PREM_PLATFORMS:
-        # RGW count is 1 if the cluster was upgraded from <= 4.4
-        # Related bug - https://bugzilla.redhat.com/show_bug.cgi?id=1857802
-        rgw_count = 2 if float(version_before_upgrade) >= 4.5 else 1
+        rgw_count = get_rgw_count(
+            float(upgrade_version), True, float(version_before_upgrade)
```

Comment:
What I've heard from yesterday's meeting that the upgrade from 4.6 to 4.7 that it should scale down RGW pods to 1.  I am testing it now here:  https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-trigger-vsphere-upi-1az-rhcos-vsan-3m-3w-upgrade-ocs-auto-nightly/15/console so we can check on this cluster. I am pausing before teardown as I would like to verify also another issue when we don't se noobaa core pod after upgrade.

Reply:
Was able to confirm with this execution that the rgw count scales down to 1 with an upgrade from 4.6 to 4.7. I'll update the function logic and the unit test to match this.